app/components/Utility/TranslateWithLinks.jsx

```diff
@@ -82,12 +84,27 @@ export default class TranslateWithLinks extends React.Component {
                                     base_amount={key.value.base.amount}
                                     quote_asset={key.value.quote.asset_id}
                                     quote_amount={key.value.quote.amount}
-                                />
+                                />;
                         break;
 
                     case "asset":
+                        
                         value = this.linkToAsset(key.value);
                         break;
+                    
+                    case "translate":
+                        value = <Translate content={key.value} />;
+                        break;
+
+                    case "link":
+                        value = <Link to={key.value} data-intor={key.dataintro ? key.dataintro : null}>
```

Comment:
There's a spelling error here, the prop should be `data-intro` not `data-intor`. Also, can you make dataintro camelcase instead please? `dataIntro`

Reply:
Sorry for the obvious typo. Fixed the string formatting